app/src/main/kotlin/compiler/lowlevel/storage/DisplayStorage.kt

```diff
@@ -1,10 +1,12 @@
 package compiler.lowlevel.storage
 
-import compiler.ast.Program
 import java.io.PrintWriter
 
-class DisplayStorage(val program: Program) {
+class DisplayStorage(private val programStaticDepth: Int) {
+
+    val displayLabel: String = "display"
+
     fun writeAsm(output: PrintWriter) {
-        TODO() // Reserve memory for the display with appropriate maximum depth
+        output.write("$displayLabel: resq $programStaticDepth")
```

Comment:
assumed `section .bss` is added somewhere before, because this can be not the only thing in this section.